app/assets/javascripts/oxalis/model/binary.js

```diff
@@ -269,9 +269,11 @@ class Binary {
       );
 
       // In general, pull buckets which are not available but should be sent to the GPU
+      // Don't use -1 for ortho mode since this will make at the corner of the viewport more important than the ones in the middle
+      const missingBucketPriority = viewMode === constants.MODE_PLANE_TRACING ? 100 : -1;
```

Comment:
If this is a problem in volume mode as well (which I would expect, but it's not that easy to jump there - only if the position is changed via the input field), I would change this line to: `const missingBucketPriority = constants.MODES_PLANE.includes(viewMode) ? 100 : -1;`
But I guess it's not that important, we'll have to think of a proper solution anyways, let's hotfix this first.